tests/fixtures/input/example.txt

```diff
@@ -1 +1,2 @@
 {{'whooo' | size}}{%comment%}What happens{%endcomment%} {%if num < numTwo%}wat{%else%}wot{%endif%} {%if num > numTwo%}wat{%else%}wot{%endif%}
+{% include tests/fixtures/additional/default.liquid %}
```

Comment:
If you want you can instead add another set of tests under  `tests/fixtures/{input,output}/include.txt` that check things like if included files get parsed and evaluated properly.